src/routes/transactions/mappers/common/twap-order.mapper.spec.ts

```diff
@@ -680,6 +786,7 @@ describe('TwapOrderMapper', () => {
   });
 
   it('should map the TWAP order if source apps are restricted and a part order fullAppData matches any of the allowed apps', async () => {
+    configurationService.set('swaps.maxNumberOfParts', 2);
```

Comment:
This test began failing without this.

Reply:
It's weird, I can't replicate the failure, maybe it's a flaky test for some reason. Or maybe the value is set on another test as the configuration is not cleaned between the tests if I'm not wrong. Anyway, I'm good with setting this explictly here.